kritis-charts/templates/kritis-server-deployment.yaml

```diff
@@ -23,7 +23,10 @@ spec:
       - name: {{ .Values.image.name }}
         image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
         imagePullPolicy: {{ .Values.image.pullPolicy }}
-        args: ["--tls-cert-file=/var/tls/cert", "--tls-key-file=/var/tls/key"]
+        args: ["--tls-cert-file=/var/tls/cert",
+               "--tls-key-file=/var/tls/key",
+               "--cron-interval=1m",
```

Comment:
Are we actually checking every minute, or is this for integration testing? If it's just for testing we could add the interval to [values-int.yaml](https://github.com/grafeas/kritis/blob/master/integration/values-int.yaml)